src/halmos/sevm.py

```diff
@@ -1261,6 +1261,11 @@ def call(
 
         caller = ex.prank.lookup(ex.this, to)
 
+        orig_code = ex.code.copy()
+        orig_storage = deepcopy(ex.storage)
+        orig_balance = deepcopy(ex.balance)
+        orig_log = deepcopy(ex.log)
```

Comment:
note to self: I have an idea to get rid of all these deepcopies, I'll get on it soon